demisto_sdk/commands/content_graph/parsers/pack.py

```diff
@@ -212,6 +200,16 @@ def categories(self):
     def use_cases(self):
         return [capital_case(c) for c in self.pack_metadata_dict.get("useCases", [])]
 
+    @property
+    def marketplaces(self) -> List[str]:
```

Comment:
I think we can return List[MarketplaceVersions] here. I don't see a reason to convert it twice